components/misc/debugging.cpp

```diff
@@ -0,0 +1,68 @@
+#include "debugging.hpp"
+
+namespace Misc
+{
+    DebugLevel Error = 0xff;
+    DebugLevel Warning = 0xfe;
+    DebugLevel Info = 0xfd;
+    DebugLevel Default = 0;
+    DebugLevel DebugLevelMarker = Info;
```

Comment:
These would probably be better as `constexpr`, since they should be compile-time constants.